pkg/pxc/statefulset.go

```diff
@@ -53,12 +53,16 @@ func StatefulSet(sfs api.StatefulApp, podSpec *api.PodSpec, cr *api.PerconaXtraD
 	}
 
 	if cr.Spec.PMM != nil && cr.Spec.PMM.Enabled {
-		pmmC, err := sfs.PMMContainer(cr.Spec.PMM, secret, cr)
-		if err != nil {
-			return nil, fmt.Errorf("pmm container error: %v", err)
-		}
-		if pmmC != nil {
-			pod.Containers = append(pod.Containers, *pmmC)
+		if !cr.Spec.PMM.HasSecret(secret) {
+			log.Info(`pmm is disabled. "pmmserverkey" or "pmmserver" secret should be specified`)
```

Comment:
I think we need to have the following:  
`can't enable PMM: either "pmmserver" or "pmmserverkey" key should exist in the <secret_name> secret`

Reply:
@nmarukovich I am not sure how we can stop to support it e.g. PXC operator. In new PS operator we support only pmmserverkey and it is ok.

Reply:
OK, lets use the following message

Can't enable PMM: "pmmserverkey" key should exist in the <secret_name> secret.